src/index.js

```diff
@@ -97,17 +97,22 @@ function createSlots (slots, h) {
 export function shallow (component, ...args) {
   const c = { ...component }
   c.components = { ...component.components }
-  shallowRenderComponents(c)
+  shallowRenderComponents(c, ...args)
   return mount(c, ...args)
 }
 
-function shallowRenderComponents (component) {
+function shallowRenderComponents (component, depth) {
```

Comment:
This is awesome!

Thinking about it as a whole, now that depth is definable, is this still the best name for the method? shallow is now only a default due. This seems to have become `renderComponents`